### ISSUE TITLE
FluentNavMenu should allow for a custom icon

### ISSUE THREAD

**Issue by mrpmorris** [microsoft/fluentui-blazor#463]
```
                @if (Collapsible) {
                    <FluentButton Appearance="Appearance.Lightweight"
                              Class="navmenu-main-button"
                              @onclick="CollapsibleClickAsync"
                              title="@Title">
                        <FluentIcon Name="@(FluentIcons.Navigation)" Size="IconSize.Size20" />
                    </FluentButton>
                }
```

This will force the consuming app to include icons, which is something I don't wish to do.  Perhaps we could have a NavigationContent render fragment?

**Comment by vnbaaij** [comment]
I'll look at adding a content render fragment for a later version. What I can do for the next minor version is to add the icon to the set of 'library required icons'. That way you can still use it even if you set up the project to not use icons. Would that help?

**Comment by vnbaaij** [comment]
I'll look at adding a content render fragment for a later version. What I can do for the next minor version is to add the icon to the set of 'library required icons'. That way you can still use it even if you set up the project to not use icons. Would that help?

**Comment by vnbaaij** [comment]
Fixed with #465
